### ISSUE TITLE
Clarification on tieredImageNet training settings (epochs)

### ISSUE THREAD

**Issue by itsnamgyu** [asrafulashiq/dynamic-cdfsl#4]
Thank you for sharing your work with the community. I had a question about the training settings for tieredImageNet. For step 2 (dynamic distillation stage), do you use the same number of epochs (60 epochs) for tieredImageNet, as you do for miniImagetNet? This didn't seem to be explicitly stated in the paper.

Also, just to be certain, were there any other difference in hyperparameters between miniImageNet (ResNet10) and tieredImageNet (ResNet18) training for each pre-training stage?

Best,

**Comment by asrafulashiq** [comment]
Yes, same epochs. All the other parameters are same as mini-ImageNet. However, we train on 4 gpus, and, so, scale learning rate by 4 (i.e., 0.04).
